sources/packages/backend/libs/test-utils/src/factories/offering-change-request.ts

```diff
@@ -0,0 +1,31 @@
+import { EducationProgramOffering, OfferingStatus } from "@sims/sims-db";
+
+/**
+ * Create a fake offering request change.
+ * @param relations dependencies:
```

Comment:
dependencies.? (ending with period)

Reply:
@dheepak-aot I really didn't find any issue with the `:` or`,` or `.` when we are expanding the details of a param object in the next lines. we do have some similar comments in our repo. devs let me know your thoughts.

Reply:
Neither I @ann-aot . It is just that we never had that idea agreed among devs.
It's minor but we can check once now and take it forward.

@andrewsignori-aot @guru-aot @andrepestana-aot @sh16011993 ?

Reply:
@dheepak-aot I believe our idea (or what we agreed upon) was never to use always `.`. it was to use a proper readable English sentence (starting with caps, `,`,`.`,`:` etc.. whenever wherever needed). Maybe that's the reason, we never had to discuss or agree upon a scenario like this, at least for me, that was my understanding.

Reply:
To be clear, period(.) is a part of proper readable English. As you mentioned, if we are considering starting with caps, the comment here doesn't start with caps.
Coming to the point, I am not talking about usage of `.` or `:`. Rather I am talking about the idea of using introduction text for options properties where `:` is appropriate for describing options as we do now.

Because that I am seeing both ways 1. `@param options ` commented with short summary ending with period (instead of introduction ending with `:` ) and it's properties are commented with it's own description and  2. the other way which is done here. Wanted to follow one way consistently. 

I would recommend jumping on to a call and wrap up discussions instead for the benefit of time.

Reply:
I am going to assume both the ways work and wrap this point here.

Reply:
@dheepak-aot I have no idea, why this simple idea/ conversation is going this far. If you still want to discuss this and devs are available we can have a quick sync up, not sure, if it is worth it

> @dheepak-aot I believe our idea (or what we agreed upon) was never to use always `.`. it was to use a proper readable English sentence (starting with caps, `,`,`.`,`:` etc.. whenever wherever needed). Maybe that's the reason, we never had to discuss or agree upon a scenario like this, at least for me, that was my understanding.

Reply:
As I mentioned in above comment, I really don't want to spend team's time about this as it is very trivial. It is just that I am not in same page with what has been discussed.